src/commons/components/page-navbar/NavbarDropDownOptions.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div
+    @click="selectOption"

```

Comment:
This will make it dropdown hide even if we click white area between options in dropdown.